back/engines/commercial/analytics/lib/tasks/analytics_tasks.rake

```diff
@@ -51,6 +51,29 @@ namespace :analytics do
       end
     end
   end
+
+  desc 'Populate locale dimensions from tenant config'
+  task :populate_locale_dimension, %i[host] => [:environment] do |_t, _args|
+    # Apartment::Tenant.switch(args[:host].tr('.', '_')) do
+    Apartment::Tenant.switch('localhost') do
```

Comment:
Or is it only for localhost and dev purposes?

Reply:
Think I've just left that in for dev purposes. Need to sort that